include/cuco/detail/static_multiset/static_multiset_ref.inl

```diff
@@ -499,6 +499,27 @@ class operator_impl<
     auto const& ref_ = static_cast<ref_type const&>(*this);
     return ref_.impl_.count(group, key);
   }
+
+  /**
+   * @brief Counts the occurrence of a given key contained in multiset
+   *
+   * @note If the given key has no match, the first rank of the group will
+   * return 1, other threads will return 0.
+   *
+   * @tparam ProbeKey Probe key type
+   *
+   * @param group The Cooperative Group used to perform group count
+   * @param key The key to count for
+   *
+   * @return Number of occurrences found by the current thread
+   */
+  template <typename ProbeKey>
+  __device__ size_type count_outer(cooperative_groups::thread_block_tile<cg_size> const& group,
```

Comment:
I see there is a `count_outer` which takes a CG tile but no single-thread version. Is this intentional?